src/test/java/org/example/adapter/out/database/AccountingOfficeDatabaseTest.java

```diff
@@ -0,0 +1,131 @@
+package org.example.adapter.out.database;
+
+
+import lombok.extern.slf4j.Slf4j;
+import org.example.DataCreator;
+import org.example.adapter.out.database.configuration.DatabaseHibernateConfig;
+import org.example.adapter.out.database.repository.CustomerDatabaseStorage;
+import org.example.adapter.out.database.repository.InvoiceDatabaseStorage;
+import org.example.application.InsertCostInvoiceService;
+import org.example.application.InsertIncomeInvoiceService;
+import org.example.application.InsertInvoiceService;
+import org.example.domain.NIP;
+import org.example.domain.customer.Customer;
+import org.example.domain.customer.Entrepreneurship;
+import org.example.domain.customer.EntrepreneurshipForm;
+import org.example.domain.customer.TaxPayments.IndustryType;
+import org.example.domain.customer.TaxPayments.LumpSumTax;
+import org.example.domain.invoice.CostInvoice;
+import org.example.domain.invoice.IncomeInvoice;
+import org.example.domain.invoice.Invoice;
+import org.example.port.out.CustomerRepository;
+import org.example.port.out.InvoiceRepository;
+import org.junit.jupiter.api.*;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@Slf4j
+public class AccountingOfficeDatabaseTest {
+    private CustomerRepository customerRepository;
+    private InvoiceRepository invoiceRepository;
+    private InsertInvoiceService insertInvoiceService;
+    private
+
+    @BeforeEach
+    void initClasses(){
```

Comment:
do jakiej bazy sie tutaj laczysz? masz cos lokalnie postawione?

Reply:
I had a local database set up -> In the next part of the project going to use migration and docker to store my database in there.
Resolved